applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -112,6 +112,7 @@ def GetDefaultSettings(cls):
             "displacement_control": false,
             "reform_dofs_at_each_step": false,
             "line_search": false,
+            "use_old_stiffness_in_first_iteration": false,
```

Comment:
what happens if this is used in combination with a linear analysis?

Reply:
it works without any problem. in that case it would be exact (note however that we would need to change the linear_strategy which actually would not be a bad idea)